### DISCUSSION TITLE
Mojo and jax

### DISCUSSION THREAD

**Discussion by Nightbringers** [modular/modular#861] [Q&A]
Is Mojo a replacement for Jax, or can they work together?  And which one is faster?

**Comment by lattner** [comment]
No, Mojo is a programming language like Python.  Jax is a library.  Mojo can talk to jax though :)

**Reply by ghost** [reply]
Hi Chris, How can Mojo talk to jax? Is it the same as in Python by importing Jax?

**Reply by bitsnaps** [reply]
@lattner Does [flax](https://github.com/google/flax) need a port in order to works with Mojo?

**Reply by Sharktheone** [reply]
Normally you can just import such python dependencies with the python interface. https://docs.modular.com/mojo/manual/python/
